x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -0,0 +1,445 @@
+                                                {model.os === 'Linux' && model.arch === 'amd64' ? (
+                                                  <FormattedMessage
+                                                    id="xpack.ml.trainedModels.addModelFlyout.intelLinuxLabel"
+                                                    defaultMessage="Intel and Linux optimized"
+                                                  />
+                                                ) : (
+                                                  <FormattedMessage
+                                                    id="xpack.ml.trainedModels.addModelFlyout.crossPlatformLabel"
+                                                    defaultMessage="Cross platform"
+                                                  />
+                                                )}
+                                              </b>
+                                            </EuiText>
+                                          </header>
+                                          <EuiText size={'s'} color={'subdued'}>
+                                            {model.model_id}
+                                          </EuiText>
+                                        </EuiFlexItem>
+                                        {model.recommended ? (
+                                          <EuiFlexItem grow={false}>
+                                            <EuiToolTip
+                                              content={
+                                                <FormattedMessage
+                                                  id="xpack.ml.trainedModels.modelsList.recommendedDownloadContent"
+                                                  defaultMessage="Recommended ELSER model version for your cluster's hardware configuration"
+                                                />
+                                              }
+                                            >
+                                              <EuiBadge color="hollow">
+                                                <FormattedMessage
+                                                  id="xpack.ml.trainedModels.addModelFlyout.recommendedDownloadLabel"
+                                                  defaultMessage="Recommended"
+                                                />
+                                              </EuiBadge>
+                                            </EuiToolTip>
+                                          </EuiFlexItem>
+                                        ) : null}
+                                      </EuiFlexGroup>
+                                    }
+                                    name={model.model_id}
+                                    value={model.model_id}
+                                    checked={model.model_id === selectedModelId}
+                                    onChange={() => setSelectedModelId(model.model_id)}
```

Comment:
Nitty nit, for consistency
```suggestion
                                    onChange={setSelectedModelId.bind(null, model.model_id)}
```